src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Folding/RazorCodeBlockFoldingProvider.cs

```diff
@@ -5,12 +5,13 @@
 using System.Collections.Immutable;
 using System.Threading;
 using System.Threading.Tasks;
+using Microsoft.AspNetCore.Razor.Language;
 using Microsoft.AspNetCore.Razor.LanguageServer.Extensions;
 using Microsoft.VisualStudio.LanguageServer.Protocol;
 
 namespace Microsoft.AspNetCore.Razor.LanguageServer.Folding;
 
-internal sealed class RazorCodeBlockFoldingProvider : IRazorFoldingRangeProvider
+internal sealed partial class RazorCodeBlockFoldingProvider : IRazorFoldingRangeProvider
```

Comment:
Nit: not partial any more

Reply:
Is github lying to me, or is this still partial for no reason? Not sure why this file would have needed to change at all